src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -1,11 +1,11 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TELEGRAM;
```

Comment:
The import order is triggering the checkstyle checks. on windows we can use the shortcut ctrl-alt-o to organize the imports, not sure about mac